docs/Config.md

```diff
@@ -75,3 +75,45 @@ The secrets in this example are dummy values and won't work in an actual app.
   }
 }
 ```
+
+### Running over HTTPS
+
+You need to change change the `protocol` to `https`, and tell the server where
+to find the certificate and the private key (`crt_file` and `key_file`). You'll
+need to add these values to both the `asset_server` and the `app_server`.
+
+```json
+{
+  "api": {
+    "protocol": "https",
+    "host": "www.api.dev.meetup.com",
+    "timeout": 10,
+    "root_url": "https://www.api.dev.meetup.com"
+  },
+  "asset_server": {
+    "protocol": "https",
+    "host": "beta2.dev.meetup.com",
+    "port": 0,
+    "crt_file":"/path/to/certificate.crt",
```

Comment:
no - I was thinking more about the prod setup. If/when prod static files are served from the app server, we'll probably want to revisit using the WebpackDevServer middleware in dev instead of the standalone CLI-based tool just to keep the dev setup more consistent with prod, but yeah that would need to be sorted out before the idea of an `asset_server` could go away entirely.